docs/3.0.0-beta.x/getting-started/quick-start.md

```diff
@@ -46,24 +47,64 @@ Navigate to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin
   - Click **Continue**.
 - Click the **Save** button and wait for Strapi to restart.
 
-## 5. Manage and add data to Content Type
+## 5. Create another new Content type
+
+Navigate back to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin/plugins/content-type-builder), in the left-hand menu.
+
+- Click the **"+ Add Content Type"** button.
+- Enter `category`.
+- Click **"+ Add New Field"**
```

Comment:
This line is not needed since the modal for choosing an attribute type automatically opens after the step above.

Reply:
Yes. I noticed it.  This UX is so much smoother in Beta than Alpha.  This has been edited to reflect the new UX.  thank you.